frontend/src/features/admin-form/common/AdminFormLayout.tsx

```diff
@@ -25,12 +25,14 @@ export const AdminFormLayout = (): JSX.Element => {
   const { formId } = useParams()
   if (!formId) throw new Error('No formId provided')
 
-  const { data: { siteBannerContent, adminBannerContent } = {} } = useEnv()
+  // TODO: Revert back to non-react banners post-migration.
```

Comment:
create issue and add the issue number here? maybe can just link it under https://github.com/opengovsg/FormSG/issues/4279